jupyterlab-mlops/jupyter_notebook_config.py

```diff
@@ -4,6 +4,7 @@
 c.ServerApp.ip = "0.0.0.0"
 c.ServerApp.port = 8080
 c.NotebookApp.open_browser = False
+c.NotebookApp.allow_origin_pat = 'https://[0-9a-z-]+.notebooks.googleusercontent.com'
```

Comment:
Maybe we should expose it to some config file / env variable, so that it's settable when we run in non GCP environment?

Reply:
That's tricky, because neither Vetrex note Kubeflow allows to simply inject env variable into docker. But I managed to implement this conditionability in the other way - I check if metadata server is reachable. If yes and if proxy_url is provided, I set CORS to exact domain: https://github.com/getindata/jupyter-images/pull/1/files#diff-fa6a4d2474962424b732f8a1c3b9b00afeb8f46fb89aa28a0ce9dd911a02c7d9R21-R33